src/tests/modules/bson.rs

```diff
@@ -345,3 +345,177 @@ fn debug_print() {
     assert_eq!(format!("{:?}", doc), normal_print);
     assert_eq!(format!("{:#?}", doc), pretty_print);
 }
+
+#[cfg(feature = "uuid-0_8")]
+#[test]
+fn test_binary_constructors() {
+    use crate::bson::UuidRepresentation;
+    use uuid::Uuid;
+
+    let uuid = Uuid::parse_str("00112233445566778899AABBCCDDEEFF").unwrap();
+    let bin = Binary::from_uuid(uuid);
+    assert_eq!(bin.bytes, uuid.as_bytes());
+    assert_eq!(bin.subtype, BinarySubtype::Uuid);
```

Comment:
can we also add a test here that explicit specifies the standard representation?